lnbits/core/templates/admin/_tab_security.html

```diff
@@ -0,0 +1,259 @@
+            </div>
+            <div class="col-6">
+              <q-input
+                filled
+                v-model="formAllowedIPs"
+                @keydown.enter="addAllowedIPs"
+                type="text"
+                :label="$t('enter_ip')"
+                :hint="$t('allow_access_hint')"
+              >
+                <q-btn
+                  @click="addExtensionsManifest"
+                  dense
+                  flat
+                  icon="add"
+                ></q-btn>
+              </q-input>
+              <div>
+                {%raw%}
+                <q-chip
+                  v-for="allowed_ip in formData.lnbits_allowed_ips"
+                  :key="allowed_ip"
+                  removable
+                  @remove="removeAllowedIPs(allowed_ip)"
+                  color="primary"
+                  text-color="white"
+                >
+                  {{ allowed_ip }}
+                </q-chip>
+                {%endraw%}
+              </div>
+              <br />
+            </div>
+          </div>
+        </div>
+        <div class="col-12 col-md-12">
+          <p v-text="$t('rate_limiter')"></p>
+          <div class="row q-col-gutter-md">
+            <div class="col-6">
+              <q-input
+                filled
+                type="number"
+                v-model.number="formData.lnbits_rate_limit_no"
+                :label="$t('number_of_requests')"
+              ></q-input>
+            </div>
+            <div class="col-6">
+              <q-select
+                filled
+                :options="[$t('second'),$t('minute'),$t('hour')]"
+                v-model="formData.lnbits_rate_limit_unit"
+                :label="$t('time_unit')"
+              ></q-select>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <br />
+    <div>
+      <div class="row q-col-gutter-md">
+        <div class="col-12 col-md-6">
+          <q-item tag="label" v-ripple>
+            <q-item-section>
+              <q-item-label v-text="$t('enable_notifications')"></q-item-label>
+              <q-item-label
+                caption
+                v-text="$t('enable_notifications_desc')"
+              ></q-item-label>
+            </q-item-section>
+            <q-item-section avatar>
+              <q-toggle
+                size="md"
+                v-model="formData.lnbits_notifications"
+                checked-icon="check"
+                color="green"
+                unchecked-icon="clear"
+              />
+            </q-item-section>
+          </q-item>
+          <br />
+          <p
+            v-if="!formData.lnbits_notifications"
+            v-text="$t('notifications_disabled')"
+          ></p>
+          <div v-if="formData.lnbits_notifications">
+            {% include "admin/_tab_security_notifications.html" %}
+          </div>
+          <br />
+          <div>
+            <p v-text="$t('notification_source')"></p>
+            <q-input
```

Comment:
- should show a message if the `JSON` cannot be fetched

![image](https://github.com/lnbits/lnbits/assets/2951406/03b8b084-3c8f-4df5-b4d0-8bb2313d68fa)